packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimeRangeContent.tsx

```diff
@@ -163,6 +195,27 @@ export const TimeRangeContent = (props: Props) => {
         <Trans i18nKey="time-picker.range-content.apply-button">Apply time range</Trans>
       </Button>
 
+      <div className={style.buttonsContainer}>
+        <Button
+          aria-label={t('time-picker.copy-paste.tooltip-copy', 'Copy time range to clipboard')}
+          data-testid={selectors.components.TimePicker.copyTimeRange}
+          icon="file-blank"
+          variant="secondary"
+          tooltip={t('time-picker.copy-paste.tooltip-copy', 'Copy time range to clipboard')}
+          type="button"
+          onClick={onCopy}
+        />
+        <Button
+          aria-label={t('time-picker.copy-paste.tooltip-paste', 'Paste time range')}
```

Comment:
as mentioned, i don't think you need `aria-label` if you've specified a `tooltip`